modal/app.py

```diff
@@ -717,6 +730,16 @@ def f(self, x):
                     raise InvalidError("`region` and `_experimental_scheduler_placement` cannot be used together")
                 scheduler_placement = SchedulerPlacement(region=region)
 
+            # START Experimental: Container Networking
+
+            if container_networking and not _experimental_scheduler_placement:
+                scheduler_placement = SchedulerPlacement(zone="us-east-1f")
+
+            if container_networking:
+                cloud = "aws"
```

Comment:
Can we add debug logging informing about both cloud and region / AZ scheduling? I figure it might save a debugging minutes.

Reply:
https://github.com/modal-labs/modal-client/pull/2222/files ?